stdlib/Makefile.shared

```diff
@@ -64,7 +64,7 @@ INSTALL_LIBDIR=$(DESTDIR)$(LIBDIR)
 
 install::
 	cp stdlib.cma std_exit.cmo *.cmi *.cmt *.cmti *.mli *.ml \
-	  camlheader_ur \
+	  camlheader_ur ocaml_compiler_internal_params \
```

Comment:
Isn't the case that this file only gets created if you run the set-default-params script? In which case, won't this error if you *don't* run it? `$(wildcard ocaml_compiler_internal_params)` should fix it, I think.